### PR TITLE
docs: added documentation for installing on other Linux platforms

### PR DESCRIPTION
**Pull request by bcoe**

### CODE REVIEW

File: up-and-running/platforms/other.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+There are now 3 web services running on 3 ports on your VM. Make sure that these
+ports are open on your server:
+
+| Service                   | Port  |
+|-------------------------- |------ |
+| Administrator Panel       | 8800  |
+| Registry                  | 8080  |
+| Website                   | 8081  |
+
+On Centos/RHEL servers, you may need to disable `firewalld`. It's also worth checking
```
Comment (nexdrew) [inline up-and-running/platforms/other.md]: s/Centos/CentOS/

File: up-and-running/platforms/other.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+that your default [iptable rules] aren't blocking any traffic.
```
Comment (nexdrew) [inline up-and-running/platforms/other.md]: Missing link here for `[iptable rules]`?

File: up-and-running/platforms/other.md [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+## Step 2: Install npm Enterprise Itself
+
+```bash
+sudo npm i npme -g --unsafe
+```
+
+To confirm that it worked, type `npme`. If the install succeeded you should see the
+`npme help` screen, which looks like this:
+
+  ![npmo help in terminal](/gitbook/images/npmo-help.png)
```
Comment (nexdrew) [inline up-and-running/platforms/other.md]: We should update this screenshot to reference `npme` instead. Same for Azure doc.

File: up-and-running/platforms/other.md [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+_On Ubuntu:_
+
+```bash
+curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+sudo apt-get install -y nodejs
+sudo npm i -g npm@latest
+node -v && npm -v
+```
+
+_On Centos/RHEL:_
```
Comment (nexdrew) [inline up-and-running/platforms/other.md]: Minor nit: I think "Centos" should be "CentOS"

File: up-and-running/platforms/other.md [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+Simply follow the steps listed below:
+
+## Step 1: Install npm and Node.js
+
+Now that you have a server provisioned, let's install npm and Node.js
+
+_On Ubuntu:_
+
+```bash
+curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```
Comment (nexdrew) [inline up-and-running/platforms/other.md:19]: I use Node 6 (`https://deb.nodesource.com/setup_6.x`) for our AMI. ¯\_(ツ)_/¯

**Comment by nexdrew** [comment]
I think we really just need to update the screenshot, my other stupid nits you can ignore. Otherwise, LGTM

File: up-and-running/platforms/other.md (hunk 5 above)
Comment (bcoe) [reply up-and-running/platforms/other.md:19]: I think I'll stick with Node 4, which is the LTS and I've been testing with in a few places; I'm totally cool with moving over to 6.x though in future documentation.

**Pull request merged**